java/src/jmri/jmrit/display/MultiSensorIconAdder.java

```diff
@@ -105,10 +105,6 @@ protected void makeIcons(CatalogTreeNode n) {
         }
     }
 
-    /**
-     * Only called from MultiSensorIcon popup
-     *
-     */
```

Comment:
I have been consistently removing comments that make only the statement similar to "Only called from Object X or by Method Y". I don't think we want to document in a method what calls it, since contractually, there is no control that what is documented to call a method is the only things calling a method.